dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -1330,4 +1330,35 @@ public final Response update(@Context final HttpServletRequest request,
         return response;
     }
 
+    /**
+     * Delete an existing scheme
+     *
+     * @param request
+     * @return
+     */
+    @DELETE
+    @Path("/schemes/{schemeId}")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response delete(@Context final HttpServletRequest request,
+            @PathParam("schemeId") final String schemeId) {
+        final InitDataObject initDataObject = this.webResource.init(null, true, request, true, null);
+        Logger.debug(this, "Deleting scheme with id: " + schemeId);
+        Response response = null;
+        try {
+            this.workflowHelper.delete(schemeId, initDataObject.getUser());
+            response = Response.ok(new ResponseEntityView(OK)).build(); // 200
+        } catch ( DoesNotExistException e) {
+            Logger.error(this.getClass(), "Exception attempting to delete a nonexistent schema identified by : " +schemeId + ", exception message: " + e.getMessage(), e);
+            response = ExceptionMapperUtil.createResponse(e, Response.Status.BAD_REQUEST);
```

Comment:
I think we need to send a `Response.Status.NOT_FOUND` instead of a `Response.Status.BAD_REQUEST`